proto/dydxprotocol/vault/tx.proto

```diff
@@ -15,8 +15,9 @@ service Msg {
   // DepositToVault deposits funds into a vault.
   rpc DepositToVault(MsgDepositToVault) returns (MsgDepositToVaultResponse);
 
-  // UpdateParams updates the Params in state.
-  rpc UpdateParams(MsgUpdateParams) returns (MsgUpdateParamsResponse);
+  // UpdateDefaultQuotingParams updates the default quoting params in state.
```

Comment:
nit: Create new queries / handlers rather than replacing old ones.

Reply:
I'm thinking that for `MsgServer`, i should replace the old method. It's a bit too messy otherwise with UpdateParams still there (implementation, internal msg testing, etc.)